### PR TITLE
Remove worker_id calculation worker details

### PR DESCRIPTION
**Pull request by Ram-srini**
Worker id is calculated from enclave manager

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: tests/Demo.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def local_main(config):
                 input_json_obj = json.loads(input_json_str1)
                 wo_id = hex(random.randint(1, 2**64 - 1))
                 input_json_obj["params"]["workOrderId"] = wo_id
-                input_json_obj["params"]["workerId"] = worker_obj.worker_id
+                input_json_obj["params"]["workerId"] = worker_id
```
Comment (manju956) [inline tests/Demo.py:73]: How is global variable "worker_id" getting initialized?

Comment (Ram-srini) [reply tests/Demo.py:73]: It is initialized in workerRetrieve() API.

Comment (manju956) [reply tests/Demo.py:73]: Can we add a comment to specify this?

Comment (Ram-srini) [reply tests/Demo.py:73]: Temporary fix. Manoj is removing this Demo.py file.

Comment (manju956) [reply tests/Demo.py:73]: Ok

**Pull request merged**